src/kerberos.cc

```diff
@@ -154,31 +154,8 @@ Value KerberosServer::ContextCompleteGetter(const CallbackInfo& info) {
     return Boolean::New(Env(), state()->context_complete);
 }
 
-void TestMethod(const CallbackInfo& info) {
```

Comment:
I missed this in my PR that removed callback overloads - this was only used when testing defineOperation.